js/README.md

```diff
@@ -34,11 +34,11 @@ dependencies, run `eslint` and then run the tests as long as there were no
 eslint errors.
 
 Unit tests are automatically run on pull and push requests and visible at
-https://travis-ci.org/google/open-location-code.
+https://github.com/google/open-location-code/actions.
```

Comment:
This is an example of a change that should not be in this PR. Leave it be this time, but focus! It makes it easier to track changes, and makes it easier for reviewers.